myapp/spec/factories/task.rb

```diff
@@ -0,0 +1,12 @@
+FactoryBot.define do
+  factory :task do
+    title { 'Test Task' }
+    status { 0 }
+    priority { 0 }
+    user_id { 1 }
```

Comment:
we need user_type as well for the current implementation.

```suggestion
    user_id { 1 }
    user_type { 'Task' }
```